make.xsh

```diff
@@ -133,30 +133,55 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def run_local_db():
     """
```

Comment:
Is this docstring accurate? It looks like this just runs a postgres instance.

Should this function just run `docker-compose up` to start both the backup _and_ the database containers? Or is there a special reason we'd want to run _just_ the database?

If you end up keeping/changing this, can you describe the use case you were thinking this target will be for? That will be nice for when people run `make help`.

Reply:
I wondered the same. I can easily delete it but if we want to test queries its nice so that you don't have to restore or backup the database if you already have the pgdata on your system like we do but

Reply:
Let's rename this to `start_database`. We'll have it run both the database and database backup process. Can you change that and redo the docstring to describe that?